### ISSUE TITLE
[Question] Tymon\JWTAuth implementeation

### ISSUE THREAD

**Issue by umbertix** [cloudcreativity/laravel-json-api#245]
First of all thanks for this amazing library. It´s all I could ask for and even more. 😍 

I've been trying to implement the Tymon JWTAuth library for the JWT authentication with this library and so far I have everything working but I would like to be able to use the response style of the other resources.
I don't really have a model/class for my Token and if I try to use the library response `$this->reply()->content(['token' => $token])` for example I get an error `get_class() expects parameter 1 to be object, string given` which makes absolute sense since I'm actually sending a string. I'm aware that on the documentation it says that any JSON Object that needs representation would require the adaptor etc, etc. Since I don't really need it or want it and just want to be able to return a token inside the data object, is there any way to do so?

TLDR:
I want to be able to use `$this->reply()->content(['token' => $token])` for my login and refresh endpoints, without having to create an adaptor, schema and validator.

**Comment by lindyhopchris** [comment]
@Alexanderwmc there's a `CloudCreativity\LaravelJsonApi\Http\Controllers\CreatesResponses` trait that you can apply to any controller. Then you can use `$this->reply()->content(['token' => $token])` as was suggested in the TLDR above.

**Comment by lindyhopchris** [comment]
The JSON API spec says that a JSON API document (the response content) must contain at least one of the following members:

- `data`
- `meta`
- `links`

Therefore it's fine to return a response that only contains a `meta` key. That's where I'd put the token and it would mean that a media type of `application/vnd.api+json` is still valid.
